sentry.py

```
@@ -0,0 +1,101 @@
+def fast_function():
+
+    time.sleep(0.05)
+    return "done"
+
+
+# Manually call start_profiler and stop_profiler
+# to profile the code in between
+sentry_sdk.profiler.start_profiler()
+
+with sentry_sdk.start_transaction(op="process", name="Data Processing") as transaction:
+    for i in range(0, 10):
```

Ruff (0.8.2)

Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)